Model/Transaction/Refund.php

```diff
@@ -77,9 +77,26 @@ public function build(
         );
 
         if (isset($this->request['line_items'])) {
-            foreach ($this->request['line_items'] as $lineItem) {
+            $adjustmentFee = $creditmemo->getAdjustmentNegative();
+            $adjustmentRefund = $creditmemo->getAdjustmentPositive();
+
+            foreach ($this->request['line_items'] as $k => $lineItem) {
+                $lineItemSubtotal = $lineItem['unit_price'] * $lineItem['quantity'];
+                $this->request['line_items'][$k]['discount'] += ($adjustmentFee * ($lineItemSubtotal / $creditmemo->getSubtotal()));
                 $itemDiscounts += $lineItem['discount'];
```

Comment:
@ryanreeves-taxjar You may want to add a comment above this line explaining that we're _not_ including the adjustment discount in `$itemDiscounts`, since the refund-level `$discount` doesn't consider an adjustment a discount and it won't be factored in. At first I thought this could have been a potential issue since we're updating the original line items array right above but not increasing `$itemDiscounts`, but this makes sense.